casper/src/main/scala/coop/rchain/casper/blocks/proposer/BlockCreator.scala

```diff
@@ -165,13 +167,15 @@ object BlockCreator {
       preStateHash: StateHash,
       postStateHash: StateHash,
       deploys: Seq[ProcessedDeploy],
+      rejectedDeploys: Seq[ProcessedDeploy],
       systemDeploys: Seq[ProcessedSystemDeploy],
       bondsMap: Seq[Bond],
       shardId: String,
       version: Long
   ): BlockMessage = {
-    val state  = RChainState(preStateHash, postStateHash, bondsMap.toList, blockData.blockNumber)
-    val body   = Body(state, deploys.toList, systemDeploys.toList)
+    val state = RChainState(preStateHash, postStateHash, bondsMap.toList, blockData.blockNumber)
+    val body =
+      Body(state, deploys.toList, List.empty, systemDeploys.toList)
```

Comment:
should not this `List.empty` be actually `rejectedDeploys`?

Reply:
very good catch